packages/backend/src/app/project/project-entity.ts

```diff
@@ -2,14 +2,14 @@ import { EntitySchema } from 'typeorm'
 import { AppConnection, Flow, Folder, Project, TriggerEvent, User } from '@activepieces/shared'
 import { ApIdSchema, BaseColumnSchemaPart } from '../database/database-common'
 
-type ProjectSchema = {
+type ProjectSchema = Project & {
     owner: User
     flows: Flow[]
     files: File[]
     folders: Folder[]
     events: TriggerEvent[]
     appConnections: AppConnection[]
-} & Project
+}
 
 export const ProjectEntity = new EntitySchema<ProjectSchema>({
     name: 'project',
```

Comment:
I think we need to add platformId and type with their migration, please keep the migration separated as it will be for all editions